test/infrastructure/kind/mapper.go

```diff
@@ -83,22 +83,22 @@ var preBuiltMappings = []Mapping{
 	{
 		KubernetesVersion: semver.MustParse("1.33.1"),
 		Mode:              Mode0_20,
-		Image:             "kindest/node:v1.33.1@sha256:8d866994839cd096b3590681c55a6fa4a071fdaf33be7b9660e5697d2ed13002",
+		Image:             "kindest/node:v1.33.1@sha256:050072256b9a903bd914c0b2866828150cb229cea0efe5892e2b644d5dd3b34f",
```

Comment:
If I remember well from previous bumps, what we usually do is to create a new section in this file with images for the new release.

Is there a specific reason why we are changing images for 0.28 in this bump?

Reply:
+1 we should add a new section

Reply:
Hm. @fabriziopandini Or given that we just bumped 2 days ago to v0.28. Is there any reason why we couldn't essentially "skip" v0.28?

Reply:
The published image tags (apart from the sha's) are also the same. Which means the v0.28 config would be basically dead code as far as I can tell?

Reply:
Hi @sbueringer and @fabriziopandini.,
Thanks for pointing out...  As rightly understood, we've two versions of kind point to the same patch versions of kubernetes across minor releases.. 

Should it suffice to change L82 as : `// Pre-built images for Kind v0.28 and v0.29`
while essentially retaining the Version@SHA256 of the images built with kind 0.29, given that the vulnerability in containerd is patched through images built using kind-v0.29?

Please let me know your thoughts.
Thanks!

Reply:
I would prefer picking one of these options but not mixing them
1. only have a section for v0.29
2. have one section for v0.28 and one for v0.29

I'll let @fabriziopandini choose :)

Reply:
Sure!
Just a thought, if we were to use the `kind-node` images bundled with v0.28 release, the node images would essentially contain containerd-v2.1, which contain the vulnerability.. We may need to steer clear of it!

Reply:
I think that's a good reason for option 1 (basically don't even have v0.28 in the mapper)

Reply:
I've updated the PR to contain the images from kind v0.29 alone. I'll keep a tab on this thread to accommodate further changes, if any.

 
Thanks!